node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -53,13 +54,17 @@ impl EraId {
     }
 
     /// Returns an iterator over all eras that are still bonded in this one, including this one.
-    pub(crate) fn iter_bonded(&self, bonded_eras: u64) -> impl Iterator<Item = EraId> {
-        (self.0.saturating_sub(bonded_eras)..=self.0).map(EraId)
+    pub(crate) fn iter_bonded(
```

Comment:
Maybe this shouldn't be called "bonded" anymore, because it's now used for another purpose as well, where you basically pass in `3 * bonded_eras`.

Reply:
`iter_past`, maybe? And should I make it a method on the `EraSupervisor`, like the other one?